android/src/com/mapswithme/util/LocationUtils.java

```diff
@@ -107,4 +111,29 @@ public static boolean areLatLonEqual(MapObject first, MapObject second)
     return Math.abs(first.getLat() - second.getLat()) < LocationUtils.LAT_LON_EPSILON &&
            Math.abs(first.getLon() - second.getLon()) < LocationUtils.LAT_LON_EPSILON;
   }
+
+  @SuppressWarnings("deprecation")
+  public static boolean isLocationServicesTurnedOn()
```

Comment:
Пока нигде, но будет в будущем обязательно. Для показа диалога, когда не используем google play service

> On May 6, 2016, at 6:55 PM, Viktor Govako <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/util/LocationUtils.java <https://github.com/mapsme/omim/pull/3126#discussion_r62349499>:
> 
> > @@ -107,4 +111,29 @@ public static boolean areLatLonEqual(MapObject first, MapObject second)
> >      return Math.abs(first.getLat() - second.getLat()) < LocationUtils.LAT_LON_EPSILON &&
> >             Math.abs(first.getLon() - second.getLon()) < LocationUtils.LAT_LON_EPSILON;
> >    }
> > +
> > +  @SuppressWarnings("deprecation")
> > +  public static boolean isLocationServicesTurnedOn()
> А где эта функция используется?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3126/files/51d6ed31dd084fb3d3babcfc5bab306cbf88e448#r62349499>